src/MacVim/MMCoreTextView.m

```diff
@@ -1468,7 +1519,8 @@ - (void)clearBlockFromRow:(int)row1 column:(int)col1 toRow:(int)row2
 
 - (void)clearAll
 {
-    CGContextRef context = [[NSGraphicsContext currentContext] graphicsPort];
+    [self releaseLayer];
```

Comment:
I'm basically using it as a stand-in for resize code: when we're in live-resize, I need to keep the old (smaller or larger) layer around in order to draw smoothly -- so I can't toss out the layer and re-initialize it on a "formal" resize event, I have to wait until we've got draw-batch that comes back from the backend. 

I suppose I coulda also hooked into `SetTextDimensionsMsgID` or something like that, but I'd be less sure of the timing of all the various events; might be jittery or racy.